app/src/modules/content/routes/item.vue

```diff
@@ -257,7 +257,8 @@ watch(
 			return;
 		}
 
-		const targetUrl = window.location.href + (window.location.href.endsWith('/') ? 'preview' : '/preview');
+		const targetUrl = new URL(window.location.href);
+		targetUrl.pathname += '/preview';
```

Comment:
Not sure if really required, but seeing that this has been in place before and it's technically possible to access the an item via `/admin/content/{collection}/{id}/`, this feels a tad safer

```suggestion
		targetUrl.pathname += targetUrl.pathname.endsWith('/') ? 'preview' : '/preview';
```